src/main/io/osd.h

```diff
@@ -126,11 +126,15 @@ typedef enum {
     OSD_TIMER_PREC_COUNT
 } osd_timer_precision_e;
 
+#define OSD_PAGE_COUNT 3
+
 typedef enum {
-    // visible, bit 0
-    OSD_FLAG_VISIBLE = (1 << 0),
+    // visible, bits 0.2
+    OSD_FLAG_VISIBLE_PAGE_1 = (1 << 0),
+    OSD_FLAG_VISIBLE_PAGE_2 = (1 << 1),
+    OSD_FLAG_VISIBLE_PAGE_3 = (1 << 2),
```

Comment:
I would prefer to start counting with zero. 
This would also make (1<<page) easier to read.
```
OSD_FLAG_VISIBLE_PAGE_0 = (1 << 0),
OSD_FLAG_VISIBLE_PAGE_1 = (1 << 1),
OSD_FLAG_VISIBLE_PAGE_2 = (1 << 2),
```